torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -88,7 +88,11 @@ void copy_range(variable_list& out, IndexRange range, at::ArrayRef<Tensor> t) {
   TORCH_CHECK(
       range.second - range.first == t.size(),
       "inconsistent range for TensorList output");
-  std::copy(t.begin(), t.end(), out.begin() + range.first);
+  std::copy_if(
```

Comment:
This is what you were referring to above? I think it is okay without this change.

Reply:
yeah, you're right. `copy_if` was my first too optimistic approach before just creating undefined Tensors.